code/src/components/RadioBtnOption.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+
+
+export const RadioBtnOption = ({ likeOption, onlikeOptionChange}) => {
+    const likeCheese = ["yes", "no thanks", "sometimes"];
+
+    return (
+      <div className="radiobtn-box">
+        <h2>Do you like cheese?</h2>
```

Comment:
It would be nice if the h2-elements worked with the accessibility and if you put in a tabIndex="0" in the <h2> ( looks like this <h2 tabIndex="0"> hello </h2>) the accessibility will work even here! :)